Misc/NEWS.d/next/C API/2024-06-11-21-38-32.gh-issue-70278.WDE4zM.rst

```diff
@@ -0,0 +1,4 @@
+:c:func:`PyUnicode_FromFormat` no longer produces the ending ``\ufffd``
+character for truncated C string when use precision with ``%s`` and ``%V``.
+It now truncates the string before the start of truncated multibyte
+sequences.
```

Comment:
What if the truncated multibyte sequence is in the middle of the string? The second half of the string is skipped?

Reply:
It will be replaced with `\ufffd`. There are tests for this.